packages/lang/Python2/package.mk

```diff
@@ -31,7 +31,7 @@ PKG_SHORTDESC="python2: The Python2 programming language"
 PKG_LONGDESC="Python2 is an interpreted object-oriented programming language, and is often compared with Tcl, Perl, Java or Scheme."
 
 PKG_TOOLCHAIN="autotools"
-PKG_BUILD_FLAGS="-parallel"
+PKG_BUILD_FLAGS="-parallel lto-parallel"
```

Comment:
`+lto-parallel`? (I know it makes no difference, but adding the `+` makes it clear this is being enabled)